app/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM alpine:3.6
+FROM alpine:latest
```

Comment:
I would not use latest. instead please pick the latest tag, if is `3.8` set here.